src/main/java/uk/gov/pay/card/app/CardApi.java

```diff
@@ -47,7 +46,12 @@ public void initialize(Bootstrap<CardConfiguration> bootstrap) {
 
     @Override
     public void run(CardConfiguration configuration, Environment environment) {
-        environment.healthChecks().register("ping", new Ping());
+        environment.healthChecks().register("ping", new HealthCheck() {
```

Comment:
Seems appropriate use of an anonymous class since its very small and only used once. Not something for this PR as its like-for-like but wondering whether cardId just being reachable is sufficient to say its healthy (albeit its a pretty simple service)